OpenRA.Mods.Common/UtilityCommands/OutputActorSequencesCommand.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+
+namespace OpenRA.Mods.Common.UtilityCommands
+{
+	public class OutputActorSequenceCommand : IUtilityCommand
+	{
+		string IUtilityCommand.Name { get { return "--actor-sequences"; } }
+
+		bool IUtilityCommand.ValidateArguments(string[] args)
+		{
+			return args.Length == 2 || args.Length == 3;
+		}
+
+		[Desc("ACTOR-TYPE [PATH/TO/MAP]", "Display the finalized, merged MiniYaml sequence tree for the given actor type. Input values are case-sensitive.")]
+		void IUtilityCommand.Run(Utility utility, string[] args)
+		{
+			// HACK: The engine code assumes that Game.modData is set.
+			var modData = Game.ModData = utility.ModData;
+
+			var actorType = args[1];
+			string mapPath = null;
+
+			if (args.Length == 3)
+				mapPath = args[2];
+
+			var result = Utilities.GetTopLevelNodeByKey(modData, actorType,
+				manifest => manifest.Sequences,
+				map => map.SequenceDefinitions,
+				mapPath);
```

Comment:
You have the mappings just above, so might as well use `args.Length == 3 ? args[2] : null` here.